examples/scripts/countdown.js

```diff
@@ -1,5 +1,6 @@
+"use strict";
 /********************************************************************************
- * Copyright (c) 2022 Contributors to the Eclipse Foundation
+ * Copyright (c) 2021 Contributors to the Eclipse Foundation
```

Comment:
These license changes are ok? should we maybe update the ones coming from TS ? are they good?

Reply:
I noticed that also, but I think we should not bother too much. I think updating according to TS files is fine...